.travis.yml

```diff
@@ -25,10 +25,6 @@ _ADDONS_ARTIFACTS: &ADDONS_ARTIFACTS
             - $(find ./tests/_output/* -not -empty | tr "\n" ":")
             - $(find ./tests/PyzTest/*/*/_output/* -not -empty | tr "\n" ":")
             - $(find ./vendor/spryker/spryker/Bundles/*/tests/_output/* -not -empty | tr "\n" ":")
-            - /tmp/chromedriver.log.gz
```

Comment:
Can we empty this file? It should just be empty for suite IMO.

Reply:
I guess travis file changes don't have any sense for suite in any case. We don't use it

Reply:
Yeah, but we often removed it and the merge re-added it. Also keeping it empty didn't work..
We need to find a way to fully  keep it removed here.